monkey/monkey_island/cc/ui/src/components/report-components/security/PostBreach.js

```diff
@@ -30,6 +30,17 @@ const subColumns = [
 ];
 
 let renderDetails = function (data) {
+  data.forEach(pba => {
+    if (typeof pba['result'][0] === "object") {  // if `result` has more than one entry
+      let results = pba['result'];
+      let details = data.splice(data.indexOf(pba), 1);  // remove that pba from `data`
+      results.forEach(result => {  // add back those results to `data` as individual pba entries
+        let tempDetails = JSON.parse(JSON.stringify(details));
+        tempDetails[0]['result'] = result;
+        data.push(tempDetails[0]);
+      });
+    }
+  });
```

Comment:
This code shouldn't be there and I wonder if it should be on the front end at all. Can't we do it a) via mongo query? b) back end?

Reply:
Changed